README.md

```diff
@@ -12,38 +12,38 @@
 <img width="838" src="https://user-images.githubusercontent.com/204594/113578478-26912400-9623-11eb-9ff6-9bd9717462b6.png">
 </p>
 
-<sub>*(The health-bar and XP-bar are off by default, but can be enabled in the [game settings](https://github.com/diasurgical/devilutionX/wiki/DevilutionX-diablo.ini-configuration-guide). Widescreen can also be disabled if preferred)*</sub>
```

Comment:
The comma should be there. A but should be preceded by a comma.

Reply:
I agree with KP's revision. The rule is "and" or "but" is preceded by a comma if it separates two independent clauses, like if the second clause had explicitly specified a subject. A good rule of thumb is to consider whether it can be split into two separate sentences without introducing any more words.

For example, this would require a comma:
> The health-bar and XP-bar are off by default, but **they** can be enabled in the game settings.